src/prefixer.js

```diff
@@ -47,7 +47,13 @@ function getPrefixer(userAgent: ?string): InlineStylePrefixer {
   }
 
   if (!_cachedPrefixer || actualUserAgent !== _lastUserAgent) {
-    _cachedPrefixer = new InlineStylePrefixer({userAgent: actualUserAgent});
+    if (actualUserAgent === 'all') {
+      _cachedPrefixer = {
+        prefix: InlineStylePrefixer.prefixAll,
```

Comment:
I haven't thought about the `prefixedKeyframes`. That would definitely require more work.

> That would mean changing getPrefixedKeyframes to return an array, and changing the keyframes plugin to add a chunk of style for each prefix in the array (or one big chunk with all of them, more likely.

That makes sense. I have added a `prefixedKeyframes` below to fix the `flow` check.
Can we merge the PR without the `prefixedKeyframes` support? Otherwise, I will have to submit a PR to https://github.com/rofrischmann/inline-style-prefixer first and update this PR and add some test.